roles/cluster_template/tasks/main.yaml

```diff
@@ -0,0 +1,176 @@
+- name: Create common dirs
+  file:
+    path: "{{ item }}"
+    state: directory
+    mode: '0755'
+  loop:
+    - "output/{{ clusterName }}"
```

Comment:
Here (and everywhere else), `output` should probably be a variable (so that it's possible to place the generated output wherever we want, including in the appropriate cluster overlay directory).